goentri.com.the-com.json

```diff
@@ -0,0 +1,32 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "the-com",
+  "serviceName": "The.com",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up domain using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com",
+  "variableDescription": "v1 is the customer hosted domain name, sslNameVerification and sslValueVerification are to validate the certificate.",
+  "hostRequired": true,
+  "sharedProviderName": true,
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "@",
+      "pointsTo": "%v1%",
+      "ttl": 3600
+    },
+    {
+      "type": "CNAME",
+      "host": "%sslNameVerification%",
+      "pointsTo": "%sslValueVerification%",
+      "ttl": 3600
+    },
+    {
+      "type": "REDIR301",
```

Comment:
I am a bit confused by this template.
What kind of configuration would you expect?
Say setup for "example.com".
Do you expect `example.com` redirect to `www.example.com` and then CNAME on `www.example.com` + SSL validation CNAME on a host `%sslNameVerification%.example.com` pointing to `%sslValueVerification%`

Reply:
@pawel-kow From this template I'd expect (considering host=www):

Redirect example.com to www.example.com

CNAME | www | v1
CNAME | sslNameVerification.**www** | sslValueVerification

From the back and forth with Google in regards to this template we understood that when a `host` is passed it will be included on every record host. Notice also that we set `hostRequired` true.

Reply:
here add host `@`

Reply:
OK, so kind of as I expected.
The issue is that with this approach the redirect wouldn't be applied to the main domain, because `host` would be appended also here.

I added review comments to indicate needed changes.

Reply:
@pawel-kow Got it, from the docs I understood that REDIR301 record type expected one single field **target**, since this won't be a record but a URL redirect.

Our intention was to include the host & domain on the **target** for consistency.

From your proposed changes below I understand **host** is hardcoded to **www** we'd like the ability to make it variable, this way users could setup any subdomain like: **www**, **blog**, etc. Is there a way to fix the redirect issue an keeping host variable?

Reply:
> from the docs I understood that REDIR301 record type expected one single field target, since this won't be a record but a URL redirect.
The specification of REDIR301 is quite shallow, I agree. Mainly because it's an extension and not offered by every DNS provider.

Anyway, for REDIR301 parameters like `host`, `groupId` or `essential` apply as well. I will open an issue on the spec project to cover it better.

> From your proposed changes below I understand host is hardcoded to www we'd like the ability to make it variable, this way users could setup any subdomain like: www, blog, etc. Is there a way to fix the redirect issue while keeping host variable?

you can use a variable instead of `www` if it's needed

Reply:
@pawel-kow Addressed your comments above. Lmk how it looks.